src/common/components/Radio/Radio.tsx

```diff
@@ -1,20 +1,55 @@
-import * as React from 'react'
+import React, { useEffect, useState } from 'react'
+import cn from 'classnames'
 
-import * as Styles from './Radio.styles'
-
-interface RadioProps {
+export type TRadioProps = {
   isChecked: boolean
-  clickHandler: (event: React.MouseEvent<HTMLLabelElement, MouseEvent>) => void
+  clickHandler: (param: boolean) => void
+  className?: string
+  smallCircleClass?: string
   children: React.ReactNode
 }
 
-function Radio({ children, isChecked, clickHandler }: RadioProps): JSX.Element {
+const Radio = ({
+  children,
+  isChecked,
+  clickHandler,
+  className = 'w-5 h-5',
+  smallCircleClass = 'w-2 h-2',
+}: TRadioProps): JSX.Element => {
+  const [checked, setChecked] = useState<boolean>(isChecked)
+  useEffect(() => {
+    setChecked(isChecked)
+  }, [isChecked])
   return (
-    <Styles.Container className='radioContainer' onClick={clickHandler}>
-      {children}
-      <Styles.Input type='radio' checked={isChecked} onChange={() => null} />
-      <Styles.Span className='radioCheckmark' />
-    </Styles.Container>
+    <div className='flex items-center'>
+      <div
```

Comment:
radio input is better than a div: user can navigate to it using tab and toggle it using space

Reply:
fixed